bin/run-test-ci

```diff
@@ -9,8 +9,9 @@ docker::ensure_unit_test_env
 docker run --rm \
   --network test-support_default \
   -v "$(pwd)/universal-application-tool-0.0.1:/usr/src/universal-application-tool-0.0.1" \
-  -v "$(pwd)/sbt_cache/coursier:/root/.cache/coursier" \
-  -v ~/.sbt:/root/.sbt \
-  -v ~/.ivy:/root/.ivy2 \
+  -v "civiform_target:/usr/src/universal-application-tool-0.0.1/target" \
+  -v "civiform_coursier-data:/root/.cache/coursier" \
```

Comment:
ooo why are these prefaced with civiform_ instead of just being the volumes you specified below?